_config/security.yml

```diff
@@ -1,4 +1,33 @@
-SilverStripe\Security\MemberLoginForm:
+SilverStripe\Security\MemberAuthenticator\LoginForm:
   required_fields:
     - Email
-    - Password
\ No newline at end of file
+    - Password
+
+SilverStripe\Security\Security:
+  authenticators:
+    default: SilverStripe\Security\MemberAuthenticator\Authenticator
+    cms: SilverStripe\Security\MemberAuthenticator\CMSAuthenticator
```

Comment:
That would disable the option to override the default authenticator. If I implement my own authenticator and want to override, I have to awkwardly use the YML, where as
```
authenticators:
  default: My\Authenticator\Class
```
Makes more sense to me. As this would also leave the option of a custom implementation, to fall back to the default authenticator (say I lost my FB/Twitter login, 2FA thing, etc.)